hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/SuspendExecutionOnFailureTest.java

```diff
@@ -101,17 +96,16 @@ public void when_jobSuspendedByUser_then_suspensionCauseSaysSo() {
     public void when_jobSuspendedDueToFailure_then_suspensionCauseDescribeProblem() {
         // Given
         DAG dag = new DAG();
-        Vertex source = dag.newVertex("source", ListSource.supplier(singletonList(1)));
-        Vertex process = dag.newVertex("faulty",
-                new MockPMS(() -> new MockPS(() -> new MockP().setProcessError(MOCK_ERROR), MEMBER_COUNT)));
-        dag.edge(between(source, process));
+        dag.newVertex("faulty",
+                new MockPMS(() -> new MockPS(() -> new MockP().setCompleteError(MOCK_ERROR), MEMBER_COUNT)))
+                .localParallelism(1);
 
         // When
         Job job = jet().newJob(dag, jobConfig);
 
         // Then
         assertJobStatusEventually(job, JobStatus.SUSPENDED);
-        assertTrue(job.getSuspensionCause().startsWith("Due to failure:\n" +
+        assertTrue(job.getSuspensionCause().startsWith("Execution failure:\n" +
                 "com.hazelcast.jet.JetException: Exception in ProcessorTasklet{faulty#0}: " +
                 "java.lang.AssertionError: mock error"));
```

Comment:
The job can also fail in `faulty#1`. It actually fails in all processor instances, it's only a matter of chance which one is first:
```suggestion
        assertThat(job.getSuspensionCause()).matches("(?s)Execution failure:\n" +
                "com.hazelcast.jet.JetException: Exception in ProcessorTasklet\\{faulty#[0-9]}: " +
                "java.lang.AssertionError: mock error.*");
```